Cargo.lock

```diff
@@ -3083,9 +3107,9 @@ dependencies = [
  "chrono",
  "clap",
  "comfy-table",
- "frame-benchmarking",
- "frame-support",
- "frame-system",
+ "frame-benchmarking 4.0.0-dev (git+https://github.com/paritytech/substrate.git?branch=polkadot-v0.9.27)",
```

Comment:
thats why these fail, when the lockfile specifies a version and there are conflicts then this means you have an incompatibility somewhere